net-dialup/ppp/ppp-2.5.0-r5.ebuild

```
@@ -85,10 +85,11 @@ src_configure() {
 		--runstatedir="${EPREFIX}"/run
 		$(use_enable systemd)
 		$(use_with atm)
-		$(use_with pam)
+		$(usex pam --with-pam="${SYSROOT}/usr/" --without-pam)
```

The `use_with` function accepts an optional third argument to specify the value. Also, we should quote the entire substitution prevent word splitting in case the resulting path contains whitespace.
```suggestion
		"$(use_with pam pam "${SYSROOT}/usr/")"
```